### ISSUE TITLE
Voting

### ISSUE THREAD

**Issue by XinJinThePooh** [tronprotocol/wallet-cli#257]
Is there a way to vote SR with MAX numbers of votes

examples:
votewitness SRName max

Thx in advance.

**Comment by zk19862018** [comment]
No, first you need to call GetAccount Address to get the amount of TRX frozen which is the max votes available.

**Comment by XinJinThePooh** [comment]
thx a lot. Is a way that I can add something in the source code like below:

if parameter[2] = max
{
votecount= maxfrozencoin
}

I am not a programmer, but I would like to get something like this done. I can pay btw.

**Comment by zk19862018** [comment]
yes, you can add this in the source code. After this is done, you can make a PR.

**Comment by XinJinThePooh** [comment]
Thx for reply. Any chance u know where I can pay someone to do that for me?

**Comment by zk19862018** [comment]
Sorry, we dont know that.

**Comment by XinJinThePooh** [comment]
Thx anyway
